redfish-core/lib/power_supply_metrics.hpp

```diff
@@ -294,83 +294,263 @@ inline void getValues(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
 }
 
 /**
- * Systems derived class for delivering OemPowerSupplyMetrics Schema.
+ * @brief Retrieves valid input history item.
+ *
+ * Not all power supplies support the power input history. Do not provid Redfish
+ * fields for input power history if no associated endpoint matches this
+ * chassis.
+ *
+ * @param asyncResp   Pointer to object holding response data
+ * @param chassisID
```

Comment:
Missing doxygen

Reply:
Assuming you mean a description for chassisID and powerSupplyID.